### PR TITLE
August & September Updates + Reorg

### PR DESCRIPTION
**Pull request by duelinggalois**
Reorg puts updates in year folders and adds redirects for old urls. Updates for each month per usual.

### CODE REVIEW

**Review by eswarasai — APPROVED** [review]

File: docs/updates/2024/September.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+## tl;dr
+
+- Starting Audits in November
+- We had a great start on rolling out our go to market plans, much more to do and to come.
+
+## Development
+
+We are very close to code complete on the contract side and working with 0xMacro to finalize a start date in November for the audit. We will be making releases throughout the month to merge finalized versions of each code complete feature. Liquidations is the feature that will require the most effort this month to finalize, but we are going into audits regardless and communicating that the final version of liquidations may get merged at the final hour and should be reviewed last. 
+
+The front end engineers are working on adapting the front end to accommodate the changes to the core contracts and preparing the beta version that we intend to launch a beta version of the app in December.
```
Comment (1m1-github) [inline docs/updates/2024/September.md:16]:
```suggestion
The front end engineers are working on adapting the front end to accommodate the changes to the core contracts and preparing the beta version that we intend to launch in December. 
```

Comment (1m1-github) [inline docs/updates/2024/September.md:16]:
```suggestion
The front end engineers are working on adapting the front end to accommodate the changes to the core contracts and preparing the beta version that we intend to launch in December. 
```

File: docs/updates/2024/September.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+sidebar_position: 9
+---
+
+# September 2024
+
+## tl;dr
+
+- Starting Audits in November
+- We had a great start on rolling out our go to market plans, much more to do and to come.
```
Comment (1m1-github) [inline docs/updates/2024/September.md:10]: ?
```suggestion
- We had a great start on rolling out our go-to-market plans, much more to do and to come.
```

**Review by 1m1-github — APPROVED** [review]
minor comments, else looks good, not sure about sidebar positions

**Review by Uhan19 — APPROVED** [review]

**Review by yenargy — APPROVED** [review]

**Pull request merged**